docs-web-common/pom.xml

```diff
@@ -68,6 +68,27 @@
       <groupId>org.slf4j</groupId>
       <artifactId>jul-to-slf4j</artifactId>
     </dependency>
+
+    <dependency>
```

Comment:
Would it be possible to create a simple http client without openfeign and all the associated dependencies?

Reply:
@jendib are you suggesting to use resttemplates?

Reply:
@sukalpomitra OkHttp is already imported and used in `WebhookAsyncListener`. Please use that.

Reply:
hi @jendib I have implemented okHttp. Please have a look.

Reply:
hi @jendib please check the PR once you have time